concord-server/server/src/main/resources/migrations.xml

```diff
@@ -51,4 +51,18 @@
 			onUpdate="RESTRICT" referencedColumnNames="phraseId"
 			referencedTableName="phrases" />
 	</changeSet>
+	<changeSet id="4" author="wilmol">
+		<addColumn tableName="phrases">
```

Comment:
We should probably also add a lastModified timestamp to the phrases table for consistency. It would need to be set whenever the phrase record is created or updated...

Reply:
For the completed = True, it should default to NULL ?

Reply:
done phrases lastModified field, now need to test migration vs production database.